app/src/main/java/org/mifos/mobile/ui/help/HelpScreen.kt

```diff
@@ -0,0 +1,120 @@
+package org.mifos.mobile.ui.help
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.material.icons.filled.Phone
+import androidx.compose.material.icons.outlined.Mail
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.EmptyDataView
+import org.mifos.mobile.core.ui.component.FaqItemHolder
+import org.mifos.mobile.core.ui.component.MifosTextButtonWithTopDrawable
+import org.mifos.mobile.models.FAQ
+
+
+@Composable
+fun HelpScreen(
+    faqArrayList: List<FAQ?>?,
+    selectedFaqPosition: Int = -1,
+    callNow: () -> Unit,
+    leaveEmail: () -> Unit,
+    findLocations: () -> Unit,
+    navigateBack: () -> Unit,
+    searchQuery: (String) -> Unit,
+    onSearchDismiss: () -> Unit,
+    updateFaqPosition: (Int) -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+        HelpTopBar(
+            navigateBack = navigateBack,
+            searchQuery = searchQuery,
+            titleResourceId = R.string.help,
+            onSearchDismiss = onSearchDismiss
+        )
+
+        Text(
+            text = stringResource(id = R.string.faq),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 16.dp, end = 16.dp, top = 12.dp, bottom = 8.dp),
+            style = MaterialTheme.typography.titleMedium,
+            color = if (isSystemInDarkTheme()) Color.White else Color.Black
+        )
+
+        if (!faqArrayList.isNullOrEmpty() && faqArrayList.isNotEmpty()) {
```

Comment:
`!faqArrayList.isNullOrEmpty()` is enough, don't ned to add `faqArrayList.isNotEmpty()`